contracts/contracts/sbtc-deposit.clar

```diff
@@ -54,20 +55,31 @@
     )
 )
 
+(define-read-only (get-burn-header (height uint))
+    ;; (get-burn-block-info? header-hash height)
+    (get-tenure-info? burnchain-header-hash height)
```

Comment:
Oh interesting, I don't know much about `get-tenure-info?`. I was able to track it down in the [source](https://github.com/stacks-network/stacks-core/blob/b09c4ac455363293ad6bd7416fe4bad6a75ed65b/clarity/src/vm/types/mod.rs#L741-L749) but my main question is whether it returns the exact same info as `get-burn-block-info?`. I suppose it probably does, but I couldn't follow the source much to verify.